src/test.cc

```diff
@@ -15,16 +15,16 @@ limitations under the License.
 
```

Comment:
test.cc is under maintenance and I'd probably remove it. So further changes to this file is not needed.